camunda-formio-tasklist-vue/src/components/TaskList.vue

```diff
@@ -385,6 +386,7 @@ export default class Tasklist extends Mixins(TaskListMixin) {
   @Prop() private getTaskId!: string;
   @Prop() private containerHeight!: string;
   @Prop({ default: "lastName" }) userListType!: string;
+  @Prop({default: "created"}) public taskSortBy!: string
```

Comment:
Document this new Prop in README as `taskSortBy`:

https://github.com/AOT-Technologies/forms-flow-ai-extensions/blob/dev/camunda-formio-tasklist-vue/README.md#required-config-variables

Reply:
#91 documents the props for TaskList